pkg/policy/validate.go

```diff
@@ -681,7 +682,10 @@ func validateAPICall(entry kyverno.ContextEntry) error {
 		return fmt.Errorf("both configMap and apiCall are not allowed in a context entry")
 	}
 
-	if _, err := engine.NewAPIPath(entry.APICall.URLPath); err != nil {
+	// Replace all variables to prevent validation failing on variable keys.
+	urlPath := variables.ReplaceAllVars(entry.APICall.URLPath, func(s string) string { return "variable" })
```

Comment:
@MarcelMue - thanks for submitting the fix!

Can we keep the same logic but change `variable` to a more unique name? "kyvernoapicallvariable" maybe?

Reply:
Done :+1: